components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowDAO.java

```diff
@@ -39,6 +44,8 @@
 public class WorkflowDAO {
 
     private final String errorMessage = "Error when executing the SQL query ";
+    private static final Log log = LogFactory.getLog(WorkflowDAO.class);
+    boolean debugEnabled = log.isErrorEnabled();
```

Comment:
missing access modifier, do we need this flag anyway? this should be enabled by default and hence no need to check if error level is enabled by default

Reply:
Removed it and corrected it.